src/MPIMeasurements.jl

```diff
@@ -19,7 +19,12 @@ using InteractiveUtils
 using Graphics: @mustimplement
 using Scratch
 using Mmap
-using Gtk
```

Comment:
@jonschumacher: Hi Jonas: I think we should drop Gtk/Winston altogether from MPIMeasurements. It was really just there as a workaround, which is disabled anyway. So just remove it entirely on your branch.

Reply:
I actually added it again in order to have some user interaction when requesting the user to confirm that now sample is in the bore (https://github.com/MagneticParticleImaging/MPIMeasurements.jl/pull/25/commits/7ea2e569766d8239cf65be5fce5655a45a8c0028). I also came to the conclusion that this might not be the best choice when dealing with the CI issues. Ideally we should have some user interaction hooks that can handle either console input or GUI input from MPIUI. I guess I will just try to finish my rework stuff and then we should have a meeting on how to proceed.

Reply:
Ah ok, then just proceed. I just wanted to indicate that it needs not to be kept because of me.